overlord/ifacestate/ifacestate.go

```diff
@@ -212,6 +222,7 @@ func connect(st *state.State, plugSnap, plugName, slotSnap, slotName string, fla
 	// wait for prepare-plug- anyway, and a simple one-to-one wait dependency makes testing easier.
 	addTask(connectInterface)
 	prev = connectInterface
+	tasks.MarkEdge(connectInterface, ConnectTask)
```

Comment:
When reviewing I was super confused with `edge` here, thinking about graph edges because of the name. 

Btw.could use a comment:
```
// mark as the last task in connect prepare
tasks.MarkEdge(...)
```

Reply:
Done.